functions.php

```diff
@@ -135,6 +136,45 @@ private function services( $services = array() ) {
 	}
 
 	/**
+	 * Adds more data to the context variable that will be passed to the main template.
+	 *
+	 * @param array $context The associative array with data to be passed to the main template.
+	 *
+	 * @return mixed
+	 */
+	public function add_to_context( $context ) {
+		$context['data_nav_bar'] = [
+			'websites'     => $this->websites,
+			'images'       => $this->theme_images_dir,
+			'home_url'     => home_url( '/' ),
+			'act_url'      => '/act',
```

Comment:
Why do you hardcode this here?

Reply:
I didn't. It was already there (in the bottom of the file). I just moved the function up.
This file is getting big and we should clear up the code sometime soon. To be honest I do not remember if I put this there, but I will see how to make it dynamic.

Reply:
Ok. There is already a jira issue about making this dynamic.